components/combobox/combobox.jsx

```diff
@@ -154,12 +169,17 @@ const propTypes = {
 		'overflowBoundaryElement',
 		'relative',
 	]),
+	/**
+	 * Sets a maximum width that the menu will be used if `inheritWidthOf` is set to `menu`.
+	 */
+	menuMaxWidth: PropTypes.string,
```

Comment:
It would be great to add an example here `(ex- '500px')` or something, so folks know to use units like `px`.